### DISCUSSION TITLE
Turbine tilt angle GCH model

### DISCUSSION THREAD

**Discussion by LuukSengers** [NatLabRockies/floris#412] [Q&A] [answered]
Hi there,

Previously I used v2.2.2, in which tilt_angle was a standard parameters in the .json-file (never tested it though), but in gch.yaml and nrel_5MW.yaml in v3.0.1 this parameters seems to be no longer present. I did find it hardcoded in [Gauss.py L146](https://github.com/NREL/floris/blob/main/floris/simulation/wake_deflection/gauss.py#L146). Would this be the correct (or only) position to change this parameter, or am I missing something?
Just as a check: NREL5MW has a shaft tilt of -5 deg in FAST. Is it correct that this shaft tilt corresponds to the tilt_angle I mentioned here?

Lastly: I'm studying wake steering for single wake cases only. Would you recommend to work with GCH or CC?

Many thanks in advance!
Luuk

**Comment by rafmudaf** [comment, marked as answer]
Hi @LuukSengers

The tilt angle in earlier versions of FLORIS are for tilt control and primarily used to model wake steering in the vertical direction.

> NREL5MW has a shaft tilt of -5 deg in FAST. Is it correct that this shaft tilt corresponds to the tilt_angle I mentioned here?

The [OpenFAST NREL 5MW](https://github.com/OpenFAST/r-test/blob/main/glue-codes/openfast/5MW_Land_DLL_WTurb/NRELOffshrBsline5MW_Onshore_ElastoDyn.dat#L56) model does specify a shaft tilt, but the performance tables (Cp vs wind speed, Ct vs wind speed) that are used in the [FLORIS NREL 5MW](https://github.com/NREL/floris/blob/main/floris/turbine_library/nrel_5MW.yaml#L8) model are created considering the upward tilt in the OpenFAST definition. Since the tilt in the turbine definition is already captured, it would be double-counted if you change the tilt angle in the Gauss deflection model.

> Would this be the correct (or only) position to change this parameter

During the v3 effort, we intentionally left tilt-control out of the models. You may have noticed the `#TODO` in the line you referenced in the Gauss deflection model. We may add this feature again in a future release.

I hope this resolves your question, but please let us know if something isn't clear.

**Reply by LuukSengers** [reply]
Hi @rafmudaf 

Appreciate you coming back to me so quickly!

One follow up question: 
I'm mainly looking at the wind field / wake at a few rotor diameters downstream, not necessarily the power generated by the turbine. In a comparison between field measurements and LES (+ ADMR of NREL 5MW with tilt = 0deg) I found a vertical wake displacement.
I just want to make sure I use the "most correct" FLORIS setup tuning it with this field data. 

With this background info, would you recommend me to keep using tilt_angle = 0, or would you suggest another setup / changes to the code?

Thanks again!
Luuk

**Reply by rafmudaf** [reply]
Well, its a good question and I'm not sure I can fully answer, but @bayc or @paulf81 probably can.

Since the NREL 5MW performance in FLORIS comes from the model with shaft tilt included, I expect that the FLORIS wake profile and the OpenFAST (via FAST.Farm) wake would be similar. Your LES simulation probably has a few more parameters available that could impact the wake.

@paulf81 do you have any insight into this? I think there's two things: comparing FLORIS results with LES and then comparing LES+ADMR with field measurements.
